epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -380,16 +414,23 @@ TODO: Context diagram, E-Commerce, Availability, Fulfilment, Event-Broker
     - or new ones, and we switch over the downstream things progressively
 
 
-
 //TODO: event capture and all that jazz
 
+// (EJ1) * "Event Capture" and "Data Capture" are used interchangeably.  Is it supposed to be
```

Comment:
Should be event interception, I think. https://martinfowler.com/bliki/EventInterception.html

I definitely _don't_ recommend change-data capture.